api/envoy/service/ext_proc/v3/external_processor.proto

```diff
@@ -98,21 +98,15 @@ message ProcessingRequest {
     // The HTTP trailers for the request path. Unless ``async_mode`` is ``true``, the server
     // must send back a TrailerResponse message or close the stream.
     //
-    // This message is only sent if the trailers processing mode is set to ``SEND``.
-    // If there are no trailers on the original downstream request, then this message
-    // will only be sent (with empty trailers waiting to be populated) if the
-    // processing mode is set before the request headers are sent, such as
-    // in the filter configuration.
+    // This message is only sent if the trailers processing mode is set to ``SEND`` and
```

Comment:
Looks good to me for these changes.

Please also change the above line 94 to be "response body" as mentioned in issue 32029.

Reply:
Good catch.